Cargo.toml

```diff
@@ -25,28 +25,6 @@ members = [
 
 resolver = "2"
 
-default-members = [
-    "aya",
-    "aya-build",
-    "aya-log",
-    "aya-log-common",
-    "aya-log-parser",
-    "aya-obj",
-    "aya-tool",
-    "test-distro",
-    "test/integration-common",
-    # test/integration-test is omitted; including it in this list causes `cargo test` to run its
```

Comment:
what happens if you run cargo test after this PR?

Reply:
It runs all tests apart from the integrartion tests.